<?xml version="1.0"?>
<launch>     
    <param name="wheel_separation" value="0.45" />
    <param name="wheel_radius" value="0.0725" />
    <param name="ticks_per_rotation" value="819.2" />
    <param name="STMdevice" type="string" value="/dev/STM" />
    <param name="MD49device" type="string" value="/dev/MD" />

    <!--laser to base tf frequency depends on the laser update rate-->
    <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args="0.355 0 0.13275 0 0 0 /base_link /base_laser 50" required="true"/>

    <node pkg="urg_node" type="urg_node" name="hokuyo" output="screen" required="true">
        <param name="frame_id" value="base_laser" />
        <param name="angle_min" value="-2.094395102" />
        <param name="angle_max" value="2.094395102" />
        <param name="latency" value="-0.19" /> 
        <!--
        <param name="min_ang" value="-1.832595715" />
        <param name="max_ang" value="1.832595715" />
        -->
    </node>

    <node pkg="gobot_base" type="wheels" name="motor_wheels" output="screen" required="true"/>
    <node pkg="gobot_base" type="odom" name="motor_odometry" output="screen" required="true"/>
    <node pkg="gobot_base" type="twist" name="motor_twist" output="screen" required="true"/>
    <node pkg="gobot_base" type="led" name="base_led" output="screen" required="true"/>
    <node pkg="gobot_software" type="gobot_status" name="software_gobot_status" output="screen" required="true" />
</launch>
